packages/protocol-utils/src/transformer_utils.ts

```diff
@@ -134,6 +134,7 @@ export enum BridgeProtocol {
     Clipper, // Not used: Clipper is now using PLP interface
     AaveV2,
     Compound,
+    Geist,
```

Comment:
This is a 1:1 Aave V2 fork right? Then you don't need to add anything here.

This list needs to match [BridgeProtocols.sol](https://github.com/0xProject/protocol/blob/development/contracts/zero-ex/contracts/src/transformers/bridges/BridgeProtocols.sol#L26-L55) so we only add unique integrations that require a deployment :)